src/vcModals.cpp

```diff
@@ -2174,6 +2174,51 @@ void vcModals_DrawImportAnnotations(vcState *pProgramState)
   }
 }
 
+void vcModals_DrawUserGuide(vcState *pProgramState)
+{
+  if (pProgramState->openModals & (1 << vcMT_UserGuide))
+  {
+    ImGui::OpenPopup("###modalUserGuide");
+    ImGui::SetNextWindowSize(ImVec2(700, 600), ImGuiCond_FirstUseEver);
+  }
+
+  if (ImGui::BeginPopupModal("###modalUserGuide", nullptr, ImGuiWindowFlags_NoTitleBar))
+  {
+    if (pProgramState->closeModals & (1 << vcMT_UserGuide))
+      ImGui::CloseCurrentPopup();
+    else
+      pProgramState->modalOpen = true;
+
+    static const char *pGuideData = nullptr;
+
+    if (ImGui::Button("Reload") || pGuideData == nullptr)
```

Comment:
This needs to be translated.

Reply:
This needs to be wrapped in GitBuild but same same